src/scss/_cp-inline-notification.scss

```diff
@@ -87,4 +87,10 @@
       color: $white;
     }
   }
+  &.is-stability-fee-warning {
+    border-radius: 5px;
+    padding: unset !important;
```

Comment:
These are necessary to override the `!important` in the padding selector for .inline-notification